SilverSolver/assignment1/Q2.py

```diff
@@ -0,0 +1,38 @@
+class single_linked_node:
+    def __init__(self, value=None, child=None):
+        self.child = child
+        self.value = value
+
+def return_k_th_from_end_element(k, start_node):
```

Comment:
A much Pythonic interface for ordered container is the brackets notation

start_node[-k] would return the node, or directly the value.

Do you think you can make thsi work? This requires implemented the __getitem__ method on the single_linked_node.